packages/client/tests/functional/metrics-disabled/tests.ts

```diff
@@ -21,8 +21,11 @@ testMatrix.setupTestSuite(
   },
   {
     optOut: {
-      from: ['mongodb'],
-      reason: 'Metrics dont work for mongodb',
+      from: ['mongodb', 'cockroachdb'],
+      reason: `
+        Metrics dont work for mongodb
```

Comment:
Metrics are provider-independent and should work for both of those. Could you point me to the failed run?

Reply:
Fixed now, mongodb don't expose these metrics: https://github.com/prisma/prisma/pull/13507/files#diff-cfa5347424177cc489589c50fb6c439d2cfdc502a36521f07bce990549b62948R35